.travis.yml

```diff
@@ -1,6 +1,50 @@
 language: node_js
 node_js:
-  - 5
+- '5'
```

Comment:
This should be indented? Also, I thought we bumped it up to 6... it might be good to do this now?

Reply:
I'll put indent back on. Sorry missed the news that the app is now in version 6, will also do the version bump now.